src/CommunityToolkit.Maui/ReadMe.txt

```diff
@@ -0,0 +1,31 @@
+﻿.NET MAUI Community Toolkit
+
+## Initializing
+
+In order to use the .NET MAUI Community Toolkit you need to call the extension method in your `MauiProgram.cs` file as follows:
+
+```csharp
```

Comment:
This is a plain text file so this doesn't provide any extra formatting. It might be nice to find a clean way of identifying what is code

Reply:
All posts I have found suggest markdown or html files are not supported but I can have a play and see

Reply:
sadly `*.md` files don't work so we will need to stick to .txt for now